doc/tutorial/network_ring.rst

```diff
@@ -16,35 +16,18 @@ In this example, a small *network* of cells, arranged in a ring, will be created
 The cell
 ********
 
-Step **(1)** shows how a simple cell with a dendrite is created. We construct the following :term:`morphology` and label the soma and dendrite:
+Step **(1)** shows how a function that returns a simple cell with a dendrite is created.
```

Comment:
This is quite confusing. Maybe:` Step **(1)** shows a function that creates a simple cell with a dendrite.`

Reply:
:+1: